platform/macos/export/export_plugin.cpp

```diff
@@ -30,11 +30,11 @@
 
 #include "export_plugin.h"
 
-#include "../logo_svg.gen.h"
-#include "../run_icon_svg.gen.h"
 #include "codesign.h"
 #include "lipo.h"
+#include "logo_svg.gen.h"
 #include "macho.h"
+#include "run_icon_svg.gen.h"
```

Comment:
Wonder if there is an argument to be made about putting generated files separately from other includes.